minid_server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask==0.12.2
+flask-sqlalchemy==2.2
+boto3==1.4.4
```

Comment:
The original minid server uses boto 1, but I'll have another PR that updates that to boto3. I highly recommend updating.